kubeflow/mpi-job/prototypes/mpi-operator.jsonnet

```diff
@@ -0,0 +1,26 @@
+// @apiVersion 0.1
+// @name io.ksonnet.pkg.mpi-operator
+// @description MPI Operator.
+// @shortDescription MPI Operator for running MPI jobs.
+// @param name string Name to give to each of the components.
+// @optionalParam namespace string null Namespace to use for the components. It is automatically inherited from the environment if not set.
+// @optionalParam image string mpioperator/mpi-operator:latest The image for the MPI Operator.
+// @optionalParam kubectlDeliveryImage string mpioperator/kubectl-delivery:latest The image for delivering kubectl.
+// @optionalParam gpusPerNode number 8 The maximum number of GPUs per node.
+
+local k = import "k.libsonnet";
+local operator = import "kubeflow/mpi-job/mpi-operator.libsonnet";
+
+local namespace = if params.namespace != "null" then params.namespace else env.namespace;
+local name = import "param://name";
```

Comment:
You can just do
local name=params.name

params will be defined in the resulting jsonnet file
https://github.com/kubeflow/kubeflow/pull/1035/files

I think its cleaner than doing import "param:/...."